docs/001_develop/03_client-capabilities/008_interaction/dialog.mdx

```diff
@@ -0,0 +1,298 @@
+
+Anchors add different types of links to your page.
+
+Use cases:
+* Creating a hyperlink to a different page
+* Linking to a specific region of the current page
+* An email link which pre-populates the recipient field
+
+## Example
+<DialogDemo />
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+  <TabItem value="genesis">
+
+    Declaration
+    ```html
+    <rapid-dialog></rapid-dialog>
+    ```
+
+    Usage
+    ```typescript
+    @customElement({
+      name: 'my-element',
+      template: html<MyElement>`
+        <rapid-dialog ${ref('dialog')}>
+          <h2>Dialog</h2>
+          <p>This is an example dialog.</p>
+        </rapid-dialog>
+        <rapid-button @click="${(x) => x.showDialog()}"></rapid-button>
+      `,
+    })
+    export class MyElement extends GenesisElement {
+      dialog: Dialog;
+      showDialog() {
+        this.dialog.show();
+      }
+    }
+    ```
+
+  </TabItem>
+  <TabItem value="react">
+
+    Declaration
+    ```html
+    <rapid-dialog></rapid-dialog>
+    ```
+
+    Usage
+    ```typescript
+    export function MyComponent() {
+      const dialogRef = useRef(null);
+
+      return (
+        <rapid-button onClick={() => {
+          dialogRef.current?.removeAttribute('hidden');
+          dialogRef.current?.show();
+        }}>
+          Open Dialog
+        </rapid-button>
+
+        <rapid-dialog
+          ref={dialogRef}
+          id="example1"
+          class="example-dialog"
+          aria-label="Simple modal dialog"
+          modal="true"
+          hidden
+        >
+          <h2>Dialog</h2>
+          <p>This is an example dialog.</p>
+        </rapid-dialog>
+      )
+    }
+    ```
+  </TabItem>
+  <TabItem value="angular">
+
+    Declaration
+    ```html
+    <rapid-dialog></rapid-dialog>
+    ```
+
+    Usage
+    ```typescript
+    import { Component, CUSTOM_ELEMENTS_SCHEMA, ViewChild, ElementRef } from '@angular/core';
+    import { FormsModule } from '@angular/forms';
+
+    @Component({
+      selector: 'my-component',
+      template: `
+        <rapid-button (click)="openDialog()">
+          Open Dialog
+        </rapid-button>
+
+        <rapid-dialog
+          #dialogRef
+          id="example1"
+          class="example-dialog"
+          aria-label="Simple modal dialog"
+          modal="true"
+          hidden
+        >
+          <h2>Dialog</h2>
+          <p>This is an example dialog.</p>
+        </rapid-dialog>
+      `,
+      standalone: true,
+      schemas: [CUSTOM_ELEMENTS_SCHEMA],
+      imports: [FormsModule],
+    })
+    export class MyComponent {
+      @ViewChild('dialogRef') dialogRef!: ElementRef;
+
+      openDialog() {
+        this.dialogRef.nativeElement?.removeAttribute('hidden');
+        this.dialogRef.nativeElement?.show();
+      }
+    }
+    ```
+  </TabItem>
+</Tabs>
+
+
+## API
+
+### Attributes
```

Comment:
I think some attributes are missing, looking in foundation-ui dialog.ts there are attributes like show-close-icon etc.